dotnet/src/InternalUtilities/src/Schema/JsonSchemaMapper.cs

```diff
@@ -0,0 +1,831 @@
+            JsonSchemaType.Object => "object",
+            _ => MapCompositeSchemaType(schemaType),
+        };
+
+        static JsonArray MapCompositeSchemaType(JsonSchemaType schemaType)
+        {
+            var array = new JsonArray();
+            foreach (JsonSchemaType type in s_schemaValues)
+            {
+                if ((schemaType & type) != 0)
+                {
+                    array.Add(MapSchemaType(type));
+                }
+            }
+
+            return array;
+        }
+    }
+
+    private const string SchemaPropertyName = "$schema";
+    private const string RefPropertyName = "$ref";
+    private const string DescriptionPropertyName = "description";
+    private const string TypePropertyName = "type";
+    private const string FormatPropertyName = "format";
+    private const string PropertiesPropertyName = "properties";
+    private const string RequiredPropertyName = "required";
+    private const string ItemsPropertyName = "items";
+    private const string AdditionalPropertiesPropertyName = "additionalProperties";
+    private const string EnumPropertyName = "enum";
+    private const string AnyOfPropertyName = "anyOf";
+    private const string ConstPropertyName = "const";
+    private const string StjValuesMetadataProperty = "$values";
+
+    private static readonly Dictionary<Type, (JsonSchemaType SchemaType, string? Format)> s_simpleTypeInfo = new()
+    {
+        [typeof(object)] = (JsonSchemaType.Any, null),
+        [typeof(bool)] = (JsonSchemaType.Boolean, null),
+        [typeof(byte)] = (JsonSchemaType.Integer, null),
+        [typeof(ushort)] = (JsonSchemaType.Integer, null),
+        [typeof(uint)] = (JsonSchemaType.Integer, null),
+        [typeof(ulong)] = (JsonSchemaType.Integer, null),
+        [typeof(sbyte)] = (JsonSchemaType.Integer, null),
+        [typeof(short)] = (JsonSchemaType.Integer, null),
+        [typeof(int)] = (JsonSchemaType.Integer, null),
+        [typeof(long)] = (JsonSchemaType.Integer, null),
+        [typeof(float)] = (JsonSchemaType.Number, null),
+        [typeof(double)] = (JsonSchemaType.Number, null),
+        [typeof(decimal)] = (JsonSchemaType.Number, null),
+#if NET6_0_OR_GREATER
+        [typeof(Half)] = (JsonSchemaType.Number, null),
+#endif
+#if NET7_0_OR_GREATER
+        [typeof(UInt128)] = (JsonSchemaType.Integer, null),
+        [typeof(Int128)] = (JsonSchemaType.Integer, null),
+#endif
+        [typeof(char)] = (JsonSchemaType.String, null),
+        [typeof(string)] = (JsonSchemaType.String, null),
+        [typeof(byte[])] = (JsonSchemaType.String, null),
+        [typeof(Memory<byte>)] = (JsonSchemaType.String, null),
+        [typeof(ReadOnlyMemory<byte>)] = (JsonSchemaType.String, null),
+        [typeof(DateTime)] = (JsonSchemaType.String, Format: "date-time"),
+        [typeof(DateTimeOffset)] = (JsonSchemaType.String, Format: "date-time"),
+        [typeof(TimeSpan)] = (JsonSchemaType.String, Format: "time"),
```

Comment:
Would it be possible to update the implementation to the most recent version? I recently pushed some changes that fix the schema for `TimeSpan`.